velox/type/Type.h

```diff
@@ -36,9 +36,9 @@
 #include "folly/CPortability.h"
 #include "velox/common/base/ClassName.h"
 #include "velox/common/serialization/Serializable.h"
-#include "velox/type/Date.h"
 #include "velox/type/StringView.h"
 #include "velox/type/Timestamp.h"
+#include "velox/type/TimestampConversion.h"
```

Comment:
Why is this new header needed here ? It should work fine to just put it in the cpp file.

Reply:
Moved to the cpp file, thanks!